apps/files/src/components/BreadCrumbs.vue

```diff
@@ -6,6 +6,8 @@
 			:aria-label="ariaLabel(section)"
 			:title="ariaLabel(section)"
 			v-bind="section"
+			dir="auto"
+			:data-dir="section.to?.query?.dir"
```

Comment:
Do we somewhere depend on this? Meaning we could maybe get rid of it altogether.
But I think we need the `to` property:

```suggestion
			:data-dir="section.to?.query?.dir"
			:to="section.to"
```

Reply:
just checked, and it does not seem as if there really are any places that use this data attribute - so i'll remove it - it is easy to add otherwise